qiskit_aer/backends/aer_compiler.py

```diff
@@ -336,6 +337,83 @@ def compile_circuit(circuits, basis_gates=None, optypes=None):
     return AerCompiler().compile(circuits, basis_gates, optypes)
 
 
+BACKEND_RUN_ARG_TYPES = {
+    "shots": int,
+    "method": str,
+    "device": str,
+    "precision": str,
+    "max_job_size": int,
+    "max_shot_size": int,
+    "enable_truncation": bool,
+    # "executor": Executor,
+    "zero_threshold": float,
+    "validation_threshold": int,
+    "max_parallel_threads": int,
+    "max_parallel_experiments": int,
+    "max_parallel_shots": int,
+    "max_memory_mb": int,
+    "fusion_enable": bool,
+    "fusion_verbose": bool,
+    "fusion_max_qubit": int,
+    "fusion_threshold": int,
+    "accept_distributed_results": bool,
+    "memory": bool,
+    # "noise_model": NoiseModel,
```

Comment:
If the two comments are meant to be semi-documenting for consistency, it might be better to explain at the top why we don't include them - it's a bit non-obvious why they're here at the moment.